### PR TITLE
chore: fixing pr message

### PR DESCRIPTION
**Pull request by yordis**

### CODE REVIEW

File: templates/.github/workflows/pr.yaml [hunk 1]
```diff
@@ -34,10 +34,10 @@ jobs:
...
             Please use one of the following types:
 
-            - **feat:** A new feature, resulting in a MINOR version bump."
-            - **fix:** A bug fix, resulting in a PATCH version bump."
-            - **refactor:** A code change that neither fixes a bug nor adds a feature, resulting in a PATCH version bump."
-            - **chore:** Changes unrelated to the release code, resulting in no version bump."
-            - **revert:** Reverts a previous commit."
+            - **feat:** A new feature, resulting in a MINOR version bump.
+            - **fix:** A bug fix, resulting in a PATCH version bump.
+            - **refactor:** A code change that neither fixes a bug nor adds a feature, resulting in a PATCH version bump.
```
Comment (doomspork) [inline templates/.github/workflows/pr.yaml:39]: After talking to @btkostner and going down a rabbit hole, I don't think this is currently possible with release-please:

```suggestion
            - **refactor:** A code change that neither fixes a bug nor adds a feature.
```

Comment (yordis) [reply templates/.github/workflows/pr.yaml:39]: 😭 what is the version change then?

Comment (btkostner) [reply templates/.github/workflows/pr.yaml:39]: I think everything added that is not a feature or breaking change is patch. Currently no configuration to change that.

Comment (yordis) [reply templates/.github/workflows/pr.yaml:39]: We need something that doesn't cause SemVer, reason for `chore` vs. `refactor`

**Review by doomspork — APPROVED** [review]

File: templates/.github/workflows/pr.yaml (hunk 1 above)
Comment (doomspork) [reply templates/.github/workflows/pr.yaml]: @yordis I really don't think it's going to be _that_ big of an issue. If it becomes a problem we can remove everything but fix and feat from generating releases.

**Pull request merged**